src/MainWindow.vala

```diff
@@ -67,6 +67,14 @@ public class Camera.MainWindow : Hdy.ApplicationWindow {
 
         camera_view = new Widgets.CameraView ();
         camera_view.bind_property ("horizontal-flip", header_bar, "horizontal-flip", GLib.BindingFlags.BIDIRECTIONAL | GLib.BindingFlags.SYNC_CREATE);
+        camera_view.button_press_event.connect ((event) => {
+            // right click
+            if (event.button == Gdk.BUTTON_SECONDARY) {
+                header_bar.take_button.activate ();
+            }
+
+            return base.button_press_event (event);
```

Comment:
Should we be calling the base handler if already handled (by emitting another signal)?

Reply:
Thanks for your review! This is mostly a copy of https://github.com/elementary/videos/blob/4374967548c0a8a607ddd25179ec24a4645c6015/src/Window.vala#L210

Reply:
OK, lets leave it.